docs/app/pages/components/components-sections/icon/icon/icon.component.html

```diff
@@ -0,0 +1,139 @@
+    <div class="ux-icon-examples-left-container">
+        <ux-icon
+            name="attachment"
+            size="24px"
+            [flipHorizontal]="false"
+            [flipVertical]="false">
+        </ux-icon>
+
+        <ux-icon
+            name="print"
+            size="24px"
+            [flipHorizontal]="false"
+            [flipVertical]="false">
+        </ux-icon>
+
+        <ux-icon
+            name="search"
+            size="25px"
+            [flipHorizontal]="false"
+            [flipVertical]="false">
+        </ux-icon>
+    </div>
+    <div class="ux-icon-examples-right-container">
+        <ux-icon
+            name="configuration"
+            size="16px"
+            [flipHorizontal]="false"
+            [flipVertical]="false">
+        </ux-icon>
+
+        <ux-icon
+            name="configuration"
+            size="32px"
+            [flipHorizontal]="true"
+            [flipVertical]="false">
+        </ux-icon>
+
+        <ux-icon
+            name="configuration"
+            size="48px"
+            rotate="90"
+            [flipHorizontal]="false"
+            [flipVertical]="false">
+        </ux-icon>
+    </div>
+</div>
+
+<hr>
+
+<p>The <code>ux-icon</code> component can be used to dynamically display icons and allows easy control of
+    their size and orientation. Icons can be selected from the UX Aspects <a [routerLink]="['/css/icons']">
+    ux-icons </a>set. The <code>ux-icon</code> component also allows alternative icons to be loaded from a
+    custom icon set to override default icons. For example, icons for a specific project could be implemented
+    via a custom set, or an icon set created for a particular size could be used.</p>
+<p>
+    The following properties can be used to configure the component:
+</p>
+
+<uxd-api-properties tableTitle="Inputs">
+    <tr uxd-api-property name="name" type="string">
+        The name of the icon to be displayed. By default this will be one of the UX Aspects icons, unless a
+        custom icon with this name has been defined.
+    </tr>
+    <tr uxd-api-property name="size" type="string">
+        The size of the icon to be displayed. This value is in CSS units for example <code>24px</code> or
+        <code>2rem</code>.
+    </tr>
+    <tr uxd-api-property name="rotate" type="90 | 180 | 270">
+        Rotation of the icon in degrees.
+    </tr>
+    <tr uxd-api-property name="flipHorizontal" type="boolean" defaultValue="false">
+        The icon can be horizontally flipped if set to <code>true</code>. The default is <code>false</code>.
+    </tr>
+    <tr uxd-api-property name="flipVertical" type="boolean" defaultValue="false">
+        The icon can be vertically flipped if set to <code>true</code>. The default is <code>false</code>.
+    </tr>
+</uxd-api-properties>
+
+<p>The following code can be used to create the example above:</p>
+
+<uxd-snippet [content]="snippets.compiled.appHtml" language="html"></uxd-snippet>
```

Comment:
We require some CSS to create the example above, we should probably document it here too, otherwise someone copying this code would not get exactly the same result as shown above

Reply:
Updated to have tabs for code snippet